utils/include/Utilities.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "stdafx.h"
+
+namespace Utilities
+{
+	/// <summary>
+	/// Return current date and time in system
+	/// </summary>
+	/// <returns>format: dd.mm.yyyy hh:mm:ss</returns>
+	std::string GetCurrentDateAndTimeFormatted();
+
+	namespace HelperDateAndTime
```

Comment:
No needs for nested namespaces, delete this namespace or separate it from Utilities